demos/storybook/stories/selection-toolbar/with-full-config.stories.ts

```diff
@@ -0,0 +1,40 @@
+import { text, number } from '@storybook/addon-knobs';
+
+export const withFullConfig = (): any => ({
+    template: `
+       <pxb-selection-toolbar [title]="title" [subtitle]="state.selected || subtitle">
+            <button mat-icon-button pxb-icon (click)="clickPXBIcon()" aria-label="menu icon">
+                <mat-icon>menu</mat-icon>
+            </button>
+        <ng-container pxb-menu>
+            <button mat-menu-item (click)="updateSubtitle('Test Item 1', state)">Test Item 1</button>
+            <button mat-menu-item (click)="updateSubtitle('Test Item 2', state)">Test Item 2</button>
+            <button mat-menu-item (click)="updateSubtitle('Test Item 3', state)">Test Item 3</button>
+        </ng-container>
+        <div pxb-right-content>
+            <button mat-icon-button *ngIf="count > 0" (click)="clickRightContentIcon('home')" aria-label="home icon"><mat-icon>home</mat-icon></button>
+            <button mat-icon-button *ngIf="count > 1" (click)="clickRightContentIcon('work')" aria-label="work icon"><mat-icon>work</mat-icon></button>
+            <button mat-icon-button *ngIf="count > 2" (click)="clickRightContentIcon('settings')" aria-label="settings icon"><mat-icon>settings</mat-icon></button>
```

Comment:
Spacing looks off on these icons...should be 16px to the right edge (from the edge of the icon...if you're using icon buttons you may need a negative margin), and 24px between icons.